projects/event-lambdas/package.json

```diff
@@ -17,10 +17,10 @@
     "dotenv": "^8.0.0",
     "jest-circus": "^26.4.2",
     "mockdate": "^3.0.5",
-    "ts-jest": "^24.0.2",
-    "ts-node": "^8.3.0",
-    "ts-node-dev": "^1.0.0-pre.40",
-    "typescript": "^3.5.3"
+    "ts-jest": "^29.1.1",
+    "ts-node": "^10.9.1",
+    "ts-node-dev": "^2.0.0",
+    "typescript": "^5.2.2"
```

Comment:
I'm not very familiar with the `editorial-tools-user-telemetry-service` - how can we test that this doesn't break things, considering it has multiple major version bumps?